### ISSUE TITLE
Download counts not always localized

### ISSUE THREAD

**Issue by pdehaan** [mozilla/testpilot#1856]
https://testpilot.dev.mozaws.net/__version__ (6225db3c8bf60c25c2bdd981472ed9b9aaa7cb25):

```json
{
    "commit": "6225db3c8bf60c25c2bdd981472ed9b9aaa7cb25",
    "version": "",
    "source": "https://github.com/mozilla/testpilot",
    "build": "https://circleci.com/gh/mozilla/testpilot/1986"

}
```

---

Similar to https://github.com/mozilla/testpilot/issues/1823
Note that on https://testpilot.dev.mozaws.net/ the download counts are localized and shows the German "160.677 Teilnehmer", but on the experiments page (https://testpilot.dev.mozaws.net/experiments/min-vid) we show a comma separated value, "160,677" and US-formatted date.

![firefox_test_pilot](https://cloud.githubusercontent.com/assets/557895/20506023/e6dbca8e-b005-11e6-9e82-c521aeddb954.png)

![firefox_test_pilot_-_min_vid](https://cloud.githubusercontent.com/assets/557895/20506052/16195e2e-b006-11e6-857f-325df7ef6923.png)

**Comment by SoftVision-PaulOiegas** [comment]
Not all locales have the users count displayed with dot separator (eg: Swedish and Hungarian). Shouldn't this be consistent on all locales ?

**Comment by flodolo** [comment]
Not necessarily, even is Swedish and Hungarian should indeed use a dot
http://www.unicode.org/cldr/charts/30/by_type/numbers.number_formatting_patterns.html

**Comment by fzzzy** [comment]
I tested this on the latest version. It seems to be fixed. It might have been related to exceptions causing l20n to not continue localizing the page.

**Comment by SoftVision-PaulOiegas** [comment]
@fzzzy Well, the localization was fixed back in 1st of December. I reopened the issue on 7th because of the user count number separator. Some languages have just a space after 3 digits, wich is looking a bit awkward. @flodolo partially confirmed what I was asking, but no one fixed that. 
It's up to you guys, I'm not hanging on this.

Here's a screenshot of the current behavior for Hungarian [hu]:

![user count no separator](https://user-images.githubusercontent.com/17258026/27858351-97bf51bc-617d-11e7-8ebe-47861b824313.png)
